iP/src/duke/Ui.java

```diff
@@ -107,4 +107,14 @@ public String readCommand() {
         user_input = scanner.nextLine();  // Read user input
         return user_input;
     }
+
+    public void showFoundTask(ArrayList<Task> foundTasks) {
+        System.out.println("Here are the matching tasks in your list:");
+        int index = 1;
```

Comment:
Perhaps it would be more readable if you had the for-loop in the form of:
for (int i = 1; i < foundTasks.size() + 1; i++) ?